code/modules/surgery/organs/stomach/stomach_ethereal.dm

```diff
@@ -6,11 +6,34 @@
 	var/crystal_charge = ETHEREAL_CHARGE_FULL
 	///used to keep ethereals from spam draining power sources
 	var/drain_time = 0
+	var/damagemodifer = 0.5
+	low_threshold_passed = "<span class='info'>You can feel your power draining as a feeling of pins and needles fills your abdomen.</span>"
+	high_threshold_passed = "<span class='warning'>Your abdomen is filled with the feeling of shocks and electric discharges!</span>"
+	high_threshold_cleared = "<span class='info'>The electric feeling in your stomach seems to fade away, leaving behind a feeling of pins and needles</span>"
+	low_threshold_cleared = "<span class='info'>The pins and needles feeling seems to die out as your charge starts to feel far more stable.</span>"
 
 /obj/item/organ/stomach/ethereal/on_life(delta_time, times_fired)
 	. = ..()
 	adjust_charge(-ETHEREAL_CHARGE_FACTOR * delta_time)
 	handle_charge(owner, delta_time, times_fired)
+	if(damage > maxHealth)//cap battery damage
+		damage = maxHealth
+	if(damage < low_threshold)
+		return
+	adjust_charge(-0.8 * (damage / 5) * delta_time) //at 100 damage ethereals will loose 16 charge per second
```

Comment:
oh god, this sound horrible to deal with. you get defibbed, then need to spend the rest of your shift next to a cyborg recharger because the stomach repair surgery can only be initiated on stomachs with >50 damage.

Reply:
like, to put this in perspective, let's lowball the amount of charge decay from this at the lowest possible number, an extra -1.6 points of charge per delta_time. if I'm correct in assuming that 1 point of delta_time correlates to 1 second passing, this would reduce the amount of time it takes to go from ETHEREAL_CHARGE_FULL to ETHEREAL_CHARGE_NORMAL (below which you take a negative moodlet) from 20.8 minutes to **6.9 minutes**. If you use the decay rate for 20 stomach damage instead of the decay rate for 10 points of stomach damage, you'll go from full to a negative moodlet in  just **4.1 minutes**. This is assuming that you know exactly when to stop charging to prevent overcharging, btw.

If this change goes through, it will deadass become the meta for newly defibbed ethereals to immediately kill themselves to repair their stomachs.

Reply:
gonna make it take as long as brain then

Reply:
> This is assuming that you know exactly when to stop charging to prevent overcharging, btw.
> 
qol: chargers with femto manipulators will not overcharge ethereals